src/test/kotlin/ApplicationTest.kt

```diff
@@ -0,0 +1,25 @@
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class ApplicationTest {
+    @Test
+    internal fun calculator() {
```

Comment:
테스트를 만들때는 해당 테스트가 무엇을 하는 테스트인지 확실하게 했으면 좋겠어요!

```suggestion
    internal fun `빈 값이 들어오면 예외를 일으킨다`() {
```